modules/azure/sql_managedinstance.go

```diff
@@ -0,0 +1,63 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/v3.0/sql"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// GetManagedInstance is a helper function that gets the sql server object.
+// This function would fail the test if there is an error.
+func GetManagedInstance(t testing.TestingT, resGroupName string, managedInstanceName string, subscriptionID string) *sql.ManagedInstance {
+	managedInstance, err := GetManagedInstanceE(t, subscriptionID, resGroupName, managedInstanceName)
+	require.NoError(t, err)
+
+	return managedInstance
+}
+
+// GetSQLServer is a helper function that gets the sql server object.
+// This function would fail the test if there is an error.
+func GetManagedInstanceDatabase(t testing.TestingT, resGroupName string, managedInstanceName string, databaseName string, subscriptionID string) *sql.ManagedDatabase {
```

Comment:
Comment should start with `GetManagedInstanceDatabase`

Reply:
Thanks @denis256 updated it